pkg/featureflags/featureflags.go

```diff
@@ -11,6 +11,8 @@ type OptionalFeatures struct {
 
 	// RoutesV2
 	RoutesV2 bool
+
+	WASMStatsFilepath string
```

Comment:
This should ideally be the feature name instead of an attribute of the feature.

Reply:
So I was planning to set this directly to a command line flag for osm-controller, but it turns out I wasn't actually propagating that all the way through and was using basically a hardcoded value. I'll make this just a bool and keep the hardcoded path. I feel like if users want to swap out the .wasm or build their own container image, they know what they're doing enough to make that work themselves.